src/AndcultureCode.CSharp.Sitefinity.Core/Extensions/DynamicContentExtensions.cs

```diff
@@ -100,6 +130,43 @@ private static object GetRelatedContent(DynamicContent content, PropertyInfo pro
             return relatedContentMapped;
         }
 
+        private static DynamicContent SetPropertyInDataItemAsRelationship<T>(
```

Comment:
I hate to say it, but I wonder if we need an `Infrastructure` layer that has it's own set of `DynamicContentExtensions`. The main reason I say this is I'm skeptical about adding references to `Telerik.OpenAccess` to this package, as essentially every project in a Sitefinity solution is going to reference this package.

Thoughts?

Reply:
I'm fine keeping this all out of the `Core` project given that it adds the `Telerik.OpenAccess`. Would it make the most sense to just keep this in our client repository until we have more of a need for a Data layer OSS repository

Reply:
Yea, for the sake of time I think lets leave it in the client repository for now.